test/utils.mjs

```diff
@@ -26,5 +27,23 @@ export async function createAosLoader(params) {
 }
 
 export async function createAntAosLoader() {
-  return createAosLoader(ANT_EVAL_OPTIONS);
+  const handle = await AoLoader(AOS_ANT_WASM, AO_LOADER_OPTIONS);
+  // just to get the mem buffer originally
+  const evalRes = await handle(
+    null,
+    {
+      ...DEFAULT_HANDLE_OPTIONS,
+
+      Tags: [{ name: 'Action', value: 'Eval' }],
+      Data: "print('foo')",
+    },
+
+    AO_LOADER_HANDLER_ENV,
+  );
+  console.dir(evalRes, { depth: null });
+
+  return {
+    handle,
+    memory: evalRes.Memory,
+  };
```

Comment:
So these tests are now using our AOS ANT custom wasm binary, instead of the aos wasm binary fixture, meaning we dont load lua code.

There were some wonky behaviour in the tests when returning a null memory, so left in a print eval just to get the array buffer.